mlflow/paddle/__init__.py

```diff
@@ -234,6 +240,11 @@ def load_model(model_uri):
             - ``models:/<model_name>/<model_version>``
             - ``models:/<model_name>/<stage>``
 
+    :param model: Required when loading a `paddle.Model` model.
```

Comment:
```suggestion
    :param model: Required when loading a `paddle.Model` model saved with `traininig=True`.
```

I think you're missing "saved with `traininig=True`".